doc/faq/crypto-considerations.rst

```diff
@@ -55,21 +55,25 @@ This way a database administrator is not able to inject rogue password hashes.
 
 The admin password hashing is performed in *lib.crypto:hash_with_pepper*.
 
+.. _faq_crypto_audit:
+
 Audit Signing
 ~~~~~~~~~~~~~
 
-The audit log is digitally signed. (see :ref:`Audit` and :ref:`cfgfile`).
+The audit log is digitally signed. (see :ref:`audit` and :ref:`audit_parameters`).
 
 The audit log can be handled by different modules. privacyIDEA comes with an
-SQL Audit Module.
+SQL Audit Module which is enabled by default.
 
 For signing the audit log the SQL Audit Module uses the RSA keys specified
-with the values *PI_AUDIT_KEY_PUBLIC* and *PI_AUDIT_KEY_PRIVATE* in
+with the values ``PI_AUDIT_KEY_PUBLIC`` and ``PI_AUDIT_KEY_PRIVATE`` in
 :ref:`cfgfile`.
 
 By default the installer generates 2048bit RSA keys.
 
+If the validity of the private key can be assured, the config entry
```

Comment:
"If the validity of the private key can be assured" and first I did not understand, that you mean, "if you can be sure that the private key is legit or something like this". Maybe you can think of a rephrase. Might not be that necessary.

Reply:
Done in 91bcff523